services-api/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -0,0 +1,15 @@
+package io.scalecube.services;
+
+import io.scalecube.net.Address;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+
+public interface Microservices {
+
+  String id();
+
+  ServiceCall call();
+
+  Address serviceAddress();
```

Comment:
In new `Microservices.Context` call plz `address()`. We don't expose any other address => no need to designate specifically that this is _service_ address.